Cumulus 3760 guidance on cleaning execution backlog (#3738)
nasa/cumulus: docs/features/execution_payload_retention.md

Upon launch of this elasticsearch task, the cleanExecutions lambda will log (accessible from CloudWatch) the task_id needed above, along with its best guess (subject to change if you are ssh tunnelling to the es cluster etc.) of the es_endpoint and formatted curl commands

## Execution backlog cleanup

> Because many users have accumulated a substantial backlog of un-cleaned execution payloads, this lambda specifies an update_limit configuration to avoid overwhelming elasticsearch and hogging too many resources.

Reviewer: ```suggestion
To facilitate removing payloads for a large quantity of executions, this lambda specifies an update_limit configuration to avoid overwhelming elasticsearch.
```

> For backlog cleanup it is recommended the following

Reviewer: ```suggestion
For cleanup of existing execution payloads the following is recommended:
```

- set the daily_execution_payload_cleanup_schedule_expression to run this hourly: `"cron(0 * * * ? *)"`
- a conservative update_limit is 1,000,000: this has been tested to be workable on a 1 node t2.small.search cluster

> Starting with this configuration 24 million es records per day can be cleaned up. A more aggressive schedule is likely possible, but will need testing in SIT/UAT to ensure compatibility with cluster configuration.

Reviewer: ```suggestion
Starting with this configuration, 24 million Elasticsearch records per day can be cleaned up. A more aggressive schedule is likely possible, but will need testing in SIT/UAT to ensure compatibility with cluster configuration.
```

> Once backlog has been taken care of, a similar configuration should be able to run once per day and keep up with ingest rate

Reviewer: ```suggestion
Once the older executions have been taken care of, a similar configuration should be able to run once per day and keep up with ingest rate
```
